data/generalhw_scripts/flash_usb_otg.sh

```diff
@@ -14,7 +14,7 @@ destination=$2
 # Size to resize (optionnal, default to 24G)
 size=${3:-24G}
 # user
-username=$(whoami)
+username=root # $(whoami)
```

Comment:
Can you add the reason for this change in the commit message body?

Reply:
Currently, it is easier to setup the whole thing, if `root` is used everywhere in the flash script. 
I will fix this later to handle a non-root user, but for now I use `root` user to move forward on other topics. So, no real good reason. ;)

Reply:
Then at least mention this. Otherwise people are wondering about the out commented code.